### PR TITLE
Initial Implementation

### PR DESCRIPTION
**Pull request by osterman**
## what
* Reference architecture for the production account

## why
* Make it easier for newcomers to spin up environments

### CODE REVIEW

File: README.yaml [hunk 1]
```diff
@@ -0,0 +1,243 @@
+name: "prod.cloudposse.co"
+
+# Canonical GitHub repo
+github_repo: cloudposse/build-harness
```
[the remark this answers is not in the document]
Comment (osterman) [reply README.yaml]: Nope. Copy pasta. Fixed.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,57 @@
+FROM cloudposse/terraform-root-modules:0.4.5 as terraform-root-modules
```
Comment (aknysh) [inline Dockerfile]: let's parameterize the ENV vars like we did here
https://github.com/cloudposse/testing.cloudposse.co/blob/master/Dockerfile

File: codefresh.yml [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# Build a service with environment variables
+version: '1.0'
+
+steps:
+  init_variables:
+    title: Init variables
+    image: alpine
+    commands:
+      - cf_export BUILD_HARNESS_VERSION=0.4.6
```
Comment (aknysh) [inline codefresh.yml]: let's use the latest version
BUILD_HARNESS_VERSION=0.6.11

File: Dockerfile (hunk 2 above)
Comment (osterman) [reply Dockerfile]: Hrmm... you mean parameterize the `0.4.5`?

Comment (aknysh) [reply Dockerfile]: specify just a few ENV vars, and then construct the other ENV vars as we did here:
https://github.com/cloudposse/testing.cloudposse.co/blob/master/Dockerfile#L15
https://github.com/cloudposse/testing.cloudposse.co/blob/master/Dockerfile#L27
https://github.com/cloudposse/testing.cloudposse.co/blob/master/Dockerfile#L31

fewer values to manage.
I'll update it.

Comment (osterman) [reply Dockerfile]: Can you fix in new PR?

**Pull request merged**